src/modules/assessments/util.tsx

```diff
@@ -99,6 +115,42 @@ export const ASSESSMENT_COLUMNS: {
   },
 };
 
+// Assessment Details column for showing custom data elements in tables
+export const ASSESSMENT_DETAILS_COL: DataColumnDef<
+  AssessmentWithCdesFragment,
+  | GetEnrollmentAssessmentsQueryVariables
+  | GetHouseholdAssessmentsQueryVariables
+  | GetClientAssessmentsQueryVariables
+  | GetProjectAssessmentsQueryVariables
+> = {
+  header: 'Assessment Details',
+  key: 'assessmentDetails',
+  render: (assessment) => {
+    return (
+      <Stack>
+        {(assessment.customDataElements || [])
+          .filter((cde) => cde.displayHooks.includes(DisplayHook.TableSummary))
+          .map((cde) => {
+            const val = customDataElementValueAsString(cde);
+            return (
+              <Typography
+                key={cde.key}
+                variant='body2'
+                sx={textTruncationSx('300px')}
```

Comment:
```suggestion
                sx={textTruncationSx()}
```

Is this necessary? I'm seeing that truncation still works appropriately when we don't pass a max width.  With 300px, it was cutting off my text even when there was plenty of space on the table to show it. It's probably not a big deal, but would err on the side of trying to show as much text as is possible based on the screen size.